plugins/modules/identity/keycloak/keycloak_authentication.py

```diff
@@ -358,8 +369,10 @@ def main():
             # Configure the executions for the flow
             if module.check_mode:
                 module.exit_json(**result)
-            if create_or_update_executions(kc=kc, config=new_auth_repr, realm=realm):
-                result['changed'] = True
+            changed, diff = create_or_update_executions(kc=kc, config=new_auth_repr, realm=realm)
+            result['changed'] = changed
```

Comment:
```suggestion
            result['changed'] |= changed
```
Sorry missed this the first time. If `force=True` I think `create_or_update_executions` will always return `changed=True`, but this should help prevent regressions.